docs/source/tensors.rst

```diff
@@ -327,8 +327,12 @@ view of a storage and defines numeric operations on it.
    .. automethod:: log2_
    .. automethod:: log_normal_
    .. automethod:: logsumexp
+   .. automethod:: logical_and
```

Comment:
I'm curious do we not have public apis for `logical_and` before?

Reply:
I don't see any (while there are bitwise ones)